### PR TITLE
Update WORKING-GROUPS.md

### PR DESCRIPTION
**Pull request by omerbensaadon**
Adding DUX

### CODE REVIEW

File: working-groups/WORKING-GROUPS.md [hunk 1]
```diff
@@ -112,26 +112,35 @@ conventions
...
-| Document Folder            | [Folder](https://drive.google.com/drive/folders/1cMXOSiGo1kiWl0WXeGz8nV6esLetJS2f)                                                                                     |
-| Repos                      | [`knative/docs`](https://github.com/knative/docs), [`knative/website`](https://github.com/knative/website)                                                             |
-| Slack Channel              | [#docs](https://slack.knative.dev/messages/docs)                                                                                                                       |
-| Github Team WG leads       | [@knative/docs-wg-leads](https://github.com/orgs/knative/teams/docs-wg-leads/members)                                                                                  |
 
 
-| &nbsp;                                                     | Leads            | Company | Profile                                     |
-| ---------------------------------------------------------- | ---------------- | ------- | ------------------------------------------- |
+
```
Comment (rhuss) [inline working-groups/WORKING-GROUPS.md:122]: Shouldn't there be a single heading for "DUX", now there are two (Docs + UX and UX)

**Review by rhuss — COMMENTED** [review]
Looks good in general, but there are is a left over heading afais ("User Experience"). 

Also, you would need to update the ToC at the top of the document.

File: working-groups/WORKING-GROUPS.md (hunk 1 above)
Comment (omerbensaadon) [reply working-groups/WORKING-GROUPS.md:122]: ah, ok on it

File: working-groups/WORKING-GROUPS.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ The current working groups are:
 
 - [API Core](#api-core)
 - [Client](#client)
-- [Documentation](#documentation)
+- [Documentation + User Experience](#documentation-+-user-experience)
```
Comment (rhuss) [inline working-groups/WORKING-GROUPS.md:58]:
```suggestion
- [Documentation + User Experience](#documentation--user-experience)
```

I think this is the proper link syntax (just tried it)

Comment (omerbensaadon) [reply working-groups/WORKING-GROUPS.md:58]: I was wondering about this! Thank you

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**